appsmithorg/appsmith
# fix: add getMetaPropertiesMap() to code scanner

## Description
Define getMetaPropertiedsMap for code scanner. Now the widget can reset using resetWidget()

Fixes #16900

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
- Manually
  - Drop a code scanner widget on the canvas
  - Use a text widget to bind its internal property CodeScanner1.value
  - Scan a QR code and observe text widget display the stored value of the code scanner
  - Use a button widget to reset the code scanner and observe that the action is executed but the internal properties are not reset

## Checklist:
